### DISCUSSION TITLE
Change icon binary sensors window

### DISCUSSION THREAD

**Discussion by pepsonEL** [home-assistant/architecture#638] [General]
Hi. In europe we have more windows as on icon name:
mdi:window-open-variant oraz mdi:window-closed-variant

Is any chance to add other device_class for this type windows ? Many user need it... of course we can create next template but it is no good solution because we have more entity... because we have binarysensors and then binarry sensor from template.... Is any solution for it to use this icon for open and close window ?

**Comment by frenck** [comment]
Even if we added a separate device class for it, it would not solve the problem you stated.

For example, if I have a door/window sensor (e.g., one from Xiaomi) there is no way of knowing how that sensor is used. Could be some kind of window (any type), door, gate, mailbox, cat litter boxes.

For these cases, icons can be customized in the UI to fit your use case. I don't see how adding a specific device class for this case will improve the situation.
